src/Components/Theme/Theme.jsx

```diff
@@ -0,0 +1,30 @@
+import Color from "../Color/Color";
+import ColorForm from "../ColorForm/ColorForm";
+
+export default function Theme({
+  colors,
+  onAddColor,
+  onDeleteColor,
+  onUpdateColor,
+}) {
+  return (
+    <>
+      <ColorForm content={"ADD COLOR"} onAddColor={onAddColor} />
```

Comment:
You do not need curly braces `{ }` when the prop value is a string

```suggestion
      <ColorForm content="ADD COLOR" onAddColor={onAddColor} />
```